Docs: #601change topic replication factor (#739)
redpanda-data/documentation: docs/data-management/data-migration.mdx

MirrorMaker adds the prefix `redpanda1` to the names of replicated topics.

:::note

> - MirrorMaker uses the `__consumer_offsets` topic to replicate consumer offsets between clusters. This is currently not supported in Redpanda, but you can follow the progress of this issue at: https://github.com/redpanda-data/redpanda/issues/1752.

Reviewer: ```suggestion
- MirrorMaker uses the `__consumer_offsets` topic to replicate consumer offsets between clusters. This is not supported in Redpanda (you can follow the progress of this issue at: https://github.com/redpanda-data/redpanda/issues/1752).
```

:::

## Change topic replication factor

> If you need to increase or decrease the replication factor on an existing topic, run: 

Reviewer: This really needs some context--under what circumstances should users increase or decrease the replication factor on an existing topic? For example, why and when would you use a factor greater than 1? Or 2? I know at Confluent the default was 3, but that the replication factor is simply the number of nodes where data is replicated. Worth spelling it out more explicitly here. When I googled this, the most common search was "why is replication factor 3?", which tells me that users don't know the answer:
"In Kafka, replication means that data is written down not just to one broker, but to many. The replication factor is a topic setting and is specified at topic creation time. A replication factor of 3 is a commonly used replication factor as it provides the right balance between broker loss and replication overhead."

MirrorMaker adds the prefix `redpanda1` to the names of replicated topics.

:::note

> - MirrorMaker uses the `__consumer_offsets` topic to replicate consumer offsets between clusters. This is not supported in Redpanda (you can follow the progress of this issue at: https://github.com/redpanda-data/redpanda/issues/1752).

Reviewer 2: @micheleRP  this is no longer true. We added support for __consumerOffsets in 22.2 , right @mmaslankaprv, @jcsp  ?

Author: @mattschumpert this got fixed in https://github.com/redpanda-data/documentation/pull/746#event-7726142650. It'll pick it up when these merge

## Stop MirrorMaker

To stop the MirrorMaker process, use `top` to find its process ID, and then run `kill <MirrorMaker pid>`.

## Troubleshooting

> If you run into any difficulty with data migration, contact us in our [Slack](https://vectorized.io/slack) community.

Reviewer: ```suggestion
If you run into any difficulty with data migration, you can request help in the Redpanda [Slack](https://vectorized.io/slack) community.
```

## Stop MirrorMaker

To stop the MirrorMaker process, use `top` to find its process ID, and then run `kill <MirrorMaker pid>`.

## Troubleshooting

> If you run into any difficulty with data migration, you can request help in the Redpanda [Slack](https://vectorized.io/slack) community.

Reviewer 2: @micheleRP is the slack still vectorized.io?

Author: Thank you, fixed!
